pkg/store/proxy.go

```diff
@@ -421,11 +426,17 @@ func (s *ProxyStore) Series(originalRequest *storepb.SeriesRequest, srv storepb.
 		if resp.GetWarning() != "" {
 			totalFailedStores++
 			maxWarningBytes := 2000
+			shouldIgnore := strings.Contains(resp.GetWarning(), "The specified key does not exist")
 			warning := resp.GetWarning()[:min(maxWarningBytes, len(resp.GetWarning()))]
-			level.Error(s.logger).Log("msg", "Store failure with warning", "warning", warning)
+			level.Error(s.logger).Log("msg", "Store failure with warning", "warning", warning, "should_ignore", shouldIgnore)
 			// Don't have group/replica keys here, so we can't attribute the warning to a specific store.
 			s.metrics.storeFailureCount.WithLabelValues("", "").Inc()
-			if r.PartialResponseStrategy == storepb.PartialResponseStrategy_GROUP_REPLICA {
+			if shouldIgnore {
```

Comment:
should this be within the if condition here: 
```
if r.PartialResponseStrategy == storepb.PartialResponseStrategy_GROUP_REPLICA 
```

Reply:
We can do that, though all our deployments are on group_replica strategy. I'll update.